esp/services/ws_machine/ws_machineService.cpp

```diff
@@ -2944,10 +2936,19 @@ IPropertyTree* Cws_machineEx::getTargetClusterUsageReq(IEspGetTargetClusterUsage
         if (roxie.length())
             readRoxieUsageReq(roxie.str(), constEnv, targetClusterTree);
 
-        SCMStringBuffer eclAgent;
+        SCMStringBuffer eclAgent, eclccServer, eclServer, eclScheduler;
         targetClusterInfo->getAgentName(eclAgent);
         if (eclAgent.length())
             readOtherComponentUsageReq(eclAgent.str(), eqEclAgent, constEnv, targetClusterTree);
+        targetClusterInfo->getECLCCServerName(eclccServer);
```

Comment:
since a cluster can only a eclccserver or a eclserver, it's only worth having a single getEclServer() method